its/projects/DuplicateAnalyzerReferences/FakeGoogleProtobuf/SomeClass.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Google.ProtoBuf
+{
+    public class SomeClass
```

Comment:
maybe rename this `FakeGoogleProtobufClass` to make it crystal clear when it's being used?